internal/service/url_service.go

```diff
@@ -21,23 +25,61 @@ func NewURLService(repo repository.URLRepository) URLService {
 }
 
 func (s urlService) CreateShortURL(req models.ShortenRequest) (string, error) {
-	//TODO implement me
-	panic("implement me")
+	short := shortener.GeneratorShortURL()
+
+	err := s.repo.SaveURL(models.URL{ShortURL: short})
```

Comment:
you need save url struct with original url which provided in request, also id, and other fields